test/e2e/utils/test_context.go

```diff
@@ -117,6 +140,24 @@ func (t *TestContext) InstallCertManager() error {
 	return err
 }
 
+// UninstallCertManager uninstalls the cert manager bundle. If hasv1beta1CRs is true,
+// the legacy version (which uses v1alpha2 CRs) is installed.
+func (t *TestContext) UninstallCertManager(hasv1beta1CRs bool) {
+	url := t.makeCertManagerURL(hasv1beta1CRs)
+	if _, err := t.Kubectl.Delete(false, "-f", url); err != nil {
+		fmt.Fprintf(GinkgoWriter,
+			"warning: error when running kubectl delete during cleaning up cert manager: %v\n", err)
+	}
+	if _, err := t.Kubectl.Delete(false, "namespace", "cert-manager"); err != nil {
+		fmt.Fprintf(GinkgoWriter, "warning: error when cleaning up the cert manager namespace: %v\n", err)
+	}
+}
+
+const (
+	prometheusOperatorVersion = "0.33"
+	prometheusOperatorURL     = "https://raw.githubusercontent.com/coreos/prometheus-operator/release-%s/bundle.yaml"
```

Comment:
wdyt about we keep all const on the top of this file?

Reply:
Usually I would do that, but these variables are only relevant to two methods in the entire file. Ditto for cert-manager variables.

Reply:
IMO the const centralized on top is a good practice that should be always followed. 
But lets no block it because of this nit.